man/tpm2_clearlock.1.md

```diff
@@ -0,0 +1,69 @@
+tpm2_clearlock 1 "DECEMBER 2017" tpm2-tools
+==================================================
+
+NAME
+----
+
+tpm2_clearlock(1) - lock/unlock access to the clear operation.
+
+SYNOPSIS
+--------
+
+`tpm2_clearlock` [OPTIONS]
+
+DESCRIPTION
+-----------
+
+tpm2_clearlock(1) - allows a user to enable (unlock) or disable (lock)
+access to the TPM2_Clear() operation. If the lockout password option
+is missing, assume NULL.
+
+OPTIONS
+-------
+
+  * **-c**, **--clear**:
+    specifies the tool should unlock access to the clear command.
+    By default it will try to disable the clear command.
+
+  * **-p**, **--platform**:
+    specifies the tool should operate on the platform hierarchy. By default
+    it operates on the lockout hierarchy.
+
+  * **-L**, **--lockout-password**=_LOCKOUT\_PASSWORD_:
```

Comment:
s/password/passwd sorry I missed that on the previous iteration.

Reply:
I missed that too. I send another update :)